### DISCUSSION TITLE
[UserStoriesToRefine] #8 As <user> I want a website where industry partners (NGO's, industry association groups, cacao co-ops, government, etc. ), that create content, can learn about how UBC can help provide language interpretation for their conferences and online events

### DISCUSSION THREAD

**Discussion by emecas** [emecas/civic-tech-website-consultancy-pilot-translation#32] [General]
### As <user> I want a website where industry partners (NGO's, industry association groups, cacao co-ops, government, etc. ), that create content, can learn about how UBC can help provide language interpretation for their conferences and online events

**Comment by emecas** [comment]
## As a [USER NAME] I can [ACHIEVE SOMETHING], so that [VALUE STATEMENT]

## Level 2- As a cacao producer I can access videos about NGO's, so that I learn about how UBC can help provide language interpretation for their conferences and online events related to NGO's category.

**Acceptance Criteria:**

## Level 2- As a cacao producer I can access videos about industry association groups, so that I learn about how UBC can help provide language interpretation for their conferences and online events related to that  industry association groups category.

**Acceptance Criteria:**

## Level 2- As a cacao producer I can access videos about cacao co-ops, so that I learn about how UBC can help provide language interpretation for their conferences and online events related to cacao co-ops category.


**Acceptance Criteria:**

## Level 2- As a cacao producer I can access videos about government, so that I  learn about how UBC can help provide language interpretation for their conferences and online events related government category.

**Acceptance Criteria:**

**Comment by UnitedByChocolate** [comment]
Level 2- As a cacao producer I can demonstrate to NGO's, what I want to learn about. 
Acceptance Criteria:
1. I will have the ability to vote on the videos that I most value
2. I will have the opportunity to provide public feedback in the form of votes and tags that will help inform NGO's, visiting the website, as to how this educational content is received.

Level 2- As an NGO I will be able to see and assess how educational content can be delivered to cacao producers videos about industry association groups, so that I learn about how UBC can help provide language interpretation for their conferences and online events related to that industry association groups category.
Acceptance Criteria:
1. I will see the comments made by producers
2. I will be able to see the "up" and "down" reddit-style votes that cacao producers make
3. I will be able to see how videos are translated and assess the quality of translation

Level 2- As an industry association, that creates my own educational content, I can witness and validate the final results of videos that are translated. 
Acceptance Criteria:
1. I will be able to see how many times videos were viewed
2. I will be able to read testimonials of how the content has affected producer's lives

Level 2- As a , industry association I will learn about how UBC can help me provide language interpretation for their conferences and online events related government category.
Acceptance Criteria:
1. I will see easy to understand packages that I can easily purchase and start into my schedule of online events.
